Makefile

```diff
@@ -204,7 +204,7 @@ push:
 	${DOCKER_COMPOSE} -f services/docker-compose.yml push ${SERVICES_LIST}
 
 # target: pull – Pulls images from a registry
-pull:
+pull: .env
```

Comment:
a bit weird to create the .env when pulling... why?

Reply:
Every time docker-compose is called, it interpolates ``docker-compose.yml`` and display lots of errors because the environs are not present. In addition now, the environments to the container are passed by ``file_env``